src/main/java/org/broadinstitute/hellbender/tools/spark/sv/CollectInsertSizeMetricsSpark.java

```diff
@@ -93,19 +96,18 @@
               optional = true)
     public int MQPassingThreshold = 0;
 
-//    TODO
-//    @Argument(doc = "If set to non-zero value, only include reads with mate passing certain mapping quality threshold.",
-//              shortName = "Qm",
-//              fullName = "mateMQPassingThreshold",
-//              optional = true)
-//    public int mateMQPassingThreshold = 0;
+    @Argument(doc = "If set to non-zero value, only include reads with mate passing certain mapping quality threshold." +
+                    "If set to zero, reads with zero mapping quality will be included in calculating metrics.",
+              shortName = "Qm",
+              fullName = "mateMQPassingThreshold",
+              optional = true)
+    public int mateMQPassingThreshold = 0;
 
-//    @Argument(doc="The level(s) at which to accumulate metrics. Possible values are {SAMPLE, LIBRARY, READ GROUP}. " +
-//                  "Default: All reads.",
-//              shortName="LEVEL",
-//              fullName = StandardArgumentDefinitions.OUTPUT_LONG_NAME,
-//              optional = false)
-//    public Set<MetricAccumulationLevel> METRIC_ACCUMULATION_LEVEL = EnumSet.of(MetricAccumulationLevel.ALL_READS);
+    @Argument(doc="The level(s) at which to accumulate metrics. Possible values are {ALL_READS, SAMPLE, LIBRARY, READ GROUP}.",
+              shortName="LEVEL",
+              fullName = "MetricsAccumulationLevel",
+              optional = false)
+    public Set<MetricAccumulationLevel> METRIC_ACCUMULATION_LEVEL = EnumSet.of(MetricAccumulationLevel.ALL_READS);
```

Comment:
fields should be camel case and this should be plural "metricAccumulationLevels"

also change the other fields in the class to camel case